src/components/Nav/DropDown/List/List.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled, { keyframes } from "styled-components";
+
+function List({ menuMouseIn, menuMouseOut, content, isHover }) {
+  return (
+    <Wrapper>
+      <MainCategory to={content.mainLink} onMouseOver={menuMouseIn}>
+        {content.category}
+      </MainCategory>
+      <DropMenu isHover={isHover} onMouseLeave={menuMouseOut}>
+        {content.subCategory.map((content, i) => (
+          <SubCategory to={content.subLink} key={i}>
+            {content.subTitle}
+          </SubCategory>
+        ))}
+      </DropMenu>
+    </Wrapper>
+  );
+}
+
+export default List;
+
+const Wrapper = styled.div`
+  display: flex;
+  position: relative;
+  height: auto;
+  flex-direction: column;
+  align-items: space-between;
+  width: 15%;
+`;
+
+const MainCategory = styled(Link)`
+  margin-right: 70px;
+  text-decoration: none;
+  color: black;
+  font-size: 17px;
+  font-weight: 700;
+
+  &:hover {
+    cursor: pointer;
+    color: #ff5600;
+  }
+`;
+
+// const opacityChange = keyframes`
+// 	0%{
+//     opacity: 0;
+//     }
+
+//   100%{
+//     opacity: 1;
+//   }
+// `;
+
+const DropMenu = styled.div`
+  position: absolute;
+  top: 30px;
+  width: 80%;
+  font-size: 14px;
+  font-weight: 400;
+  visibility: ${props => (props.isHover ? "block" : "hidden")};
```

Comment:
```suggestion
  visibility: ${({isHover}) => (isHover ? "block" : "hidden")};
```

* 구조분해 할당으로 줄이는 것도 가능합니다

Reply:
진짜 혁신이네요,,,! 그대로 반영하겠습니다 감사합니다!!